Sources/InterposeKit/InterposeKit.swift

```diff
@@ -146,6 +150,19 @@ extension Interpose {
             try execute(newState: .prepared) { try resetImplementation() }
         }
 
+        /// Release the hook block if possible.
+        public func cleanup() {
```

Comment:
If that one is public, we need additional management; what if we make this private for now, you can only deregister if it deallocates. We can tweak rest laters.